### ISSUE TITLE
Sampler --> TripletSampler?

### ISSUE THREAD

**Issue by kunhe** [Confusezius/Deep-Metric-Learning-Baselines#7]
https://github.com/Confusezius/Deep-Metric-Learning-Baselines/blob/6b0b617c02d3303c285eb52495ce5716b57425d8/losses.py#L73

Minor suggestion: renaming Sampler to TripletSampler (or TupleSampler, etc.), so that it will not be confused with torch.utils.data.Sampler :)

**Comment by Confusezius** [comment]
That's a reasonable suggestion :). I've renamed the class to `TupleSampler` though to be compatible with non-Triplet structures s.a. npair.

**Comment by kunhe** [comment]
Thanks!
